doc/tasks.rst

```diff
@@ -596,6 +596,32 @@ You can control the verbosity by:
 * from command line, see :ref:`verbosity option<verbosity_option>`.
 
 
+delayed execution
```

Comment:
doit already uses the name `delayed` to refer to tasks that are not create before execution starts. Not sure this naming will be confusing... can you think about any other name? `lag`?